mev_inspect/schemas/arbitrage.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+
+from pydantic import BaseModel
+
+from .swaps import Swap
+
+
+class Arbitrage(BaseModel):
+    swaps: List[Swap]
+    account_address: str
+    profit_token_address: str
+    start_amount: int
+    end_amount: int
+    profit_amount: int
```

Comment:
I see `profit_amount` being assigned `end_amount - start_amount` in the impl below, do we want to keep the amount denominated in the token the arb was performed in or ETH (at the time)? previous inspect impl converted all profits to ETH for consistency but I'm open to having `profit_amount` and `profit_amount_eth` to help us with advanced queries/debugging etc.

Reply:
Yeah I think separating `profit_amount` from `profit_amount_eth` is good. Basically trying to provide the most un-opinionated data here. ETH price at the time feels like an interpretation on top.

Like maybe we have a separate function prior to the DB which adds that information.